processing/joe/joe.py

```diff
@@ -254,19 +250,22 @@ def extract_iocs(self, report):
                 self.extract_url("http", iocs, lines)
             elif prefix in [
                 "analysis.behavior.network.https.packet.item.header.line.item",
-                "analysis.behavior.network.sslhttp.packet.item.header.line.item"
+                "analysis.behavior.network.sslhttp.packet.item.header.line.item",
             ]:
                 lines += "{}\n".format(value)
                 self.extract_url("https", iocs, lines)
 
         for ioc in iocs:
             self.add_ioc(ioc)
 
-
     def extract_graph(self, report):
-        report = BeautifulSoup(report, 'html.parser')
-        graph = report.find(id='behaviorGraph')
+        graph = report.find(id="behaviorGraph")
         if graph is not None:
-            graph = graph.find('svg')
-            self.results['graph'] = str(graph)
-
+            graph = graph.find("svg")
+            self.results["graph"] = str(graph)
+
+    def extract_conf(self, report):
+        conf = report.find(id="malware-configurationColl")
+        if conf is not None:
```

Comment:
```suggestion
        if conf is not None and conf.code is not None:
```

Reply:
When Joe doesn't find any config, an error (`'NoneType' object has no attribute 'string'`) is being triggered.
This error happens because the `conf` variable contains 
`<div class="accordion-body in collapse bd" id="malware-configurationColl"><div class="joeNothingFound"><span class="nothingFoundSymbol">⊘</span>No configs have been found</div></div>` in such case